Tasks/GooglePlayReleaseV4/main.ts

```diff
@@ -198,6 +198,12 @@ async function run(): Promise<void> {
             requireTrackUpdate = true;
         }
 
+        const isDraftRelease: boolean = !userFractionSupplied && tl.getBoolInput('isDraftRelease', false);
```

Comment:
https://developers.google.com/android-publisher/api-ref/rest/v3/edits.tracks#status

Shows that the status enum has 5 possible values. I think allowing users to pass status for release might be more flexible. If they add more statuses, clients who would have used it wouldn't need to update. But non the less this is a great addition.

Reply:
Absolutely, at first I thought about doing that myself, but then I noticed the task already "hides" the Release status behind the `Rollout fraction` parameter, see [here](https://github.com/microsoft/google-play-vsts-extension/blob/ac3ff45121b54d2c93afb63cb74ccdd6081b4421/Tasks/GooglePlayReleaseV4/modules/googleutil.ts#L99), so eventually I opted against it

Reply:
yeah, saw that block, I was thinking that block has to be expanded to add the status, the userFraction alone is insufficient for determining the status of the app. But what you have also solves the problem so thats great.